esb/plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/CloneMediatorGraphicalShape.java

```diff
@@ -1,8 +1,26 @@
+/*
+ * Copyright (c) 2014-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Only the created year should be in the copyright header, not a range. Use correct licence format.